### ISSUE TITLE
Update facility card design

### ISSUE THREAD

**Issue by JonAlexandra** [alexandrainst/DanskeTestfaciliteter#29]
From:
![image](https://github.com/alexandrainst/DanskeTestfaciliteter/assets/126060595/ea58c84f-a3ca-479d-9ee4-e0972ec9332c)

To:
![image](https://github.com/alexandrainst/DanskeTestfaciliteter/assets/126060595/e78a8f86-d462-4533-99e8-610ef1400bd2)

Make sure the design of the new card is final(ish) before starting this
